plugin/entrySchema.go

```diff
@@ -47,9 +47,7 @@ func Schema(e Entry) (*EntrySchema, error) {
 				// Create a schema for root so that `stree <mountpoint>` can still display
 				// it.
 				childSchema = NewEntrySchema(root, CName(root))
-				// TODO: Namespace these to something like <root_name>::Root once
-				// https://github.com/puppetlabs/wash/issues/396 is resolved.
-				childSchema.TypeID = "__" + root.name() + "__"
+				childSchema.TypeID = namespace(root.name(), "Root")
```

Comment:
None of the tests seem to test this.

Reply:
Oh, it doesn't test for the namespace form, it uses namespace to generate the expected result.